test/test_decoders.py

```diff
@@ -1672,22 +1672,27 @@ def test_beta_cuda_interface_cpu_fallback(self):
         # to the CPU path, too.
 
         ref_dec = VideoDecoder(H265_VIDEO.path, device="cuda")
-        ref_frames = ref_dec.get_frame_at(0)
-        assert (
-            _core._get_backend_details(ref_dec._decoder)
-            == "FFmpeg CUDA Device Interface. Using CPU fallback."
-        )
+
+        # Before accessing any frames, status should be unknown
+        assert not ref_dec.cpu_fallback.status_known
+
+        ref_frame = ref_dec.get_frame_at(0)
+
+        assert "FFmpeg CUDA" in str(ref_dec.cpu_fallback)
+        assert ref_dec.cpu_fallback.status_known
+        assert bool(ref_dec.cpu_fallback)
```

Comment:
Here and everywhere else, we don't need to explicitly call `bool()`. The Pythonic way is to rely on the fact that `bool()` is essentially called automatically in contexts where it's needed (it's not necessarily how it's implemented in CPython but the idea is the same). This includes:

- `assert cond`
- `if cond`
- etc.

For example, to check whether a list is empty, the Pythonic way is to just check `if l: ...`. We don't do `if bool(l):`